addons/website_report/models/report.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+
+from openerp import api, models
+from openerp.http import request
+
+
+class Report(models.Model):
+    _inherit = 'report'
+
+    @api.v7
+    def translate_doc(self, cr, uid, doc_id, model, lang_field, template, values, context=None):
+        if request and hasattr(request, 'website'):
+            if not request.website:
```

Comment:
wrong condition. if not request.website: then request.website.get_template ? 
it should be 
if request.website:
    v = request.website.get_template(template)